src/DataFixtures/ContentFixtures.php

```diff
@@ -129,4 +141,36 @@ private function getValuesforFieldType($name, $field)
 
         return $data;
     }
+
+    private function getPresetRecords(): array
+    {
+        $records['blocks'][] = [
+            'title' => 'About',
+        ];
+        $records['blocks'][] = [
+            'title' => 'Search',
+        ];
+
+        $records['tests'][] = [
+            'selectfield' => 'bar',
+            'multiselect' => 'Michelangelo',
+            'slug' => 'title-of-the-test',
+            'title' => 'Title of the test',
+            'text_markup' => 'Text with <em>markup allowed</em>.',
+            'text_plain' => 'Text with <strong>no</strong> markup allowed.',
+        ];
+
+        return $records;
+    }
+
+    private function getPreset(string $slug): array
+    {
+        if (isset($this->presetRecords[$slug]) && ! empty($this->presetRecords[$slug])) {
```

Comment:
return $this->presetRecords->has($slug) ? $this->presetRecords->get($slug)->toArray() : null;

Reply:
I don't think that does the same as the current code.